### PR TITLE
test: testing baby tree detail view

### PR DESCRIPTION
**Pull request by Jaszkowic**

### CODE REVIEW

File: src/tests/e2e/auth/tree-detail.test.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		await expect(page.getByText("Die Gattung der Ahorne")).toBeVisible();
+		await page.getByRole("button", { name: "Baumsteckbrief" }).click();
+		await expect(page.getByText("Die Gattung der Ahorne")).not.toBeVisible();
+
+		// Check for exact age
+		await expect(page.getByText("Standalter")).toBeVisible();
+		const calculatedAge = await page.getByTestId("age").textContent();
+		expect(calculatedAge).toBe(
+			// At the year of writing this test, the tree was 2 years old.
+			// Every year, the tree gets one year older, as we all do - time is merciless.
```
Comment (raphael-arce) [inline src/tests/e2e/auth/tree-detail.test.ts:35]: 🤣🤣🤣🤣

Comment (raphael-arce) [reply src/tests/e2e/auth/tree-detail.test.ts:35]: are you okay?

File: src/i18n/en.ts [hunk 2]
```diff
@@ -258,7 +258,7 @@ Use via smartphone (mobile network) can lead to performance problems (page loads
 		adoptHint:
 			"If you regularly water the same tree, you can subscribe to it. This way you can quickly find it in your profile overview.",
 		adoptErrorMessage: "Error adopting the tree. Please try again.",
-		age: (age: number) => `${age} years`,
+		age: (age: number) => `${age === 1 ? "year" : "year"}`,
```
Comment (raphael-arce) [inline src/i18n/en.ts:261]: should the second be `years`?

**Review by raphael-arce — APPROVED** [review]
one typo question otherwise lgtm 👍

File: src/tests/e2e/auth/tree-detail.test.ts (hunk 1 above)
Comment (Jaszkowic) [reply src/tests/e2e/auth/tree-detail.test.ts:35]: 🤣 yes, thanks

**Pull request merged**